src/mtx/utils/deviceType.ts

```diff
@@ -0,0 +1,322 @@
+import getBytesFromHex from '../../utils/getBytesFromHex.js';
+import getHexFromBytes from '../../utils/getHexFromBytes.js';
+
+
+const DEVICE_TYPE_INVALID_CHAR = 'x';
+const nibbles1 = ['.', '1', '3', 'R', 'M'];
+const nibbles2 = ['.', 'A', 'G', 'R', 'T', 'D'];
+const nibbles3 = ['.', '0', '1', '2', '3', '4', '5'];
+const nibbles4 = ['.', 'A', 'B', 'C', 'D', 'E', 'F'];
+const nibbles5 = ['.', 'A', 'B', 'C', 'D', 'E', 'F', 'H', 'K', 'G'];
+const nibbles6 = ['.', '1', '2', '3', '4'];
+const nibbles7 = ['.', 'L', 'M', 'Z', 'K'];
+const nibbles8 = ['.', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+const nibbles9 = ['.', 'D', 'B', 'C', 'E', 'P', 'R', 'O', 'L', 'F', 'S', 'M', 'Y', 'G', 'N', 'U'];
+const nibbles10 = ['.', '0', '1', '2', '3', '4', '5', '6', 'P', 'R', 'L', 'E', 'G', '-', '/'];
+const nibbles11 = ['.', 'H', 'A', 'T', '0', '0', '0', '0', '0', '1', '2', '3', '4', '0', '0', '0'];
+const nibbles12 = ['.', '0', '1', '2', '3', '4', '5', '6', '7', '8', 'I', 'X', 'G', 'W', 'M', '-'];
+
+const splitDeviceTypeByte = ( byte: number ): Array<number> => [
```

Comment:
```suggestion
const splitByte = ( byte: number ): Array<number> => [
```

? as it's not very DeviceType-specific

Reply:
fixed